src/sentry/integrations/message_builder.py

```diff
@@ -85,7 +85,11 @@ def build_attachment_text(group: Group, event: GroupEvent | None = None) -> Any
     ev_metadata = obj.get_event_metadata()
     ev_type = obj.get_event_type()
 
-    if ev_type == "error":
+    if event and event.occurrence.evidence_display is not None:
```

Comment:
We may want to make a new `eventtype`. I put this conditional above the checks for the event type since we don't have one yet - in my test the type is `default` but when I was injecting `IssueOccurrence` and `IssueEvidence` data into a `NameError` I triggered locally, the type was `error` so I wasn't able to get the evidence displaying w/o moving things around.

Reply:
I guess I can't comment on a line I didn't edit but in `build_attachment_title` we sort of run into this too except it's less of a problem. Right now it's falling into the bottommost else and returning `obj.title` - I believe this is okay for a generic issue based on the schema I saw.  However, we may want to display the `GROUP_TYPE_TO_TEXT` as we do for performance issues instead, idk?

Reply:
I think that if we have an `occurrence` present on the event then we could use `occurrence.issue_title` here

Reply:
> We may want to make a new eventtype. I put this conditional above the checks for the event type since we don't have one yet - in my test the type is default but when I was injecting IssueOccurrence and IssueEvidence data into a NameError I triggered locally, the type was error so I wasn't able to get the evidence displaying w/o moving things around. Not really sure what will happen with a "real" generic issue.

Seems fine for now, maybe it's better to just rely on seeing if the `occurrence` is present rather than checking the event type